packages/sdk/src/subgraph/subgraph.ts

```diff
@@ -161,6 +165,12 @@ export class Subgraph {
         const { methodContext, requestContext } = createLoggingContext("pollingLoop");
         try {
           await this.getActiveTransactions();
+
+          const activeTxsSize = this.activeTxs.size;
+          if (activeTxsSize == 0) {
+            clearInterval(this.pollingLoop!);
+            this.pollingLoop = undefined;
+          }
```

Comment:
Why do we only stop polling if there are no active transactions, wouldn't that result in some weird behavior?